uktrade/data-hub-frontend
# Added the ExportWinsMovingBanner to NavBar

## Description of change

This PR adds the "Important: Export Wins is moving to Data Hub on 3 April 2024" banner to all pages right underneath the main navbar.

## Test instructions

1. Go to any Data Hub page
2. You should see a banner saying "Important: Export Wins is moving to Data Hub on 3 April 2024" right underneath the main navbar.
3. The link in the banner should go to https://data-services-help.trade.gov.uk/data-hub/updates/announcements/export-wins-is-moving-to-data-hub/

## Screenshots

### Before

<img width="1279" alt="Screenshot 2024-03-22 at 12 07 49" src="https://github.com/uktrade/data-hub-frontend/assets/2333157/bf5a63d4-0783-42ac-bd0c-a04a17d427b0">

### After

<img width="1280" alt="Screenshot 2024-03-22 at 12 07 30" src="https://github.com/uktrade/data-hub-frontend/assets/2333157/4759a269-a756-4db3-aba3-1b1f54a17bfd">
